addons/web/static/src/views/fields/placeholder_image/placeholder_image_field.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <templates id="template" xml:space="preserve">
+    <t t-name="web.PlaceholderImageField" t-inherit="web.ImageField" t-inherit-mode="primary">
+        <xpath expr="//div[hasclass('position-relative')]/div" position="attributes">
```

Comment:
We should add a hooking class in the base template instead, this is very reliant on dom structure and styling.